src/index.js

```diff
@@ -51,14 +60,7 @@ client.on("ready", () => {
 /* ________________ MAIN MESSAGE LISTENER ________________ */
 
 client.on("message", (message) => {
-	if (!message.guild) return; // Ignores DMs
-
-	if (message.author.bot) {
-		if (message.type === "PINS_ADD") message.delete();
-		return; // Ignores messages from bots
-	}
-	if (message.type === "PINS_ADD") return; // Ignores PIN messages
-
+	if (isMessageIgnored()) return;
```

Comment:
Awesome! Thanks for working on this. ✨ 

Quick suggestion, your function looks like it might need to have the `message` object passed to it on line 63 as well.